content/en/blog/2024/docs-localized.md

```diff
@@ -0,0 +1,39 @@
+---
+title: OpenTelemetry website goes multilingual!
+linkTitle: Multilingual website
+date: 2024-08-21
+author: >-
+  [Severin Neumann](https://github.com/svrnm) (Cisco),
+issue: 4863
+sig: Comms
+cSpell:ignore: Comms
+---
+
+We are happy to announce that the OpenTelemetry website is available in
+multiple languages! Localization teams have already started to translate website
+pages into [Portuguese](/pt), [Chinese](/zh), [Japanese](/ja) and [Spanish](/es).
+
+The OpenTelemetry project has grown to include contributors and users from all
+over the world. Making the website available in multiple languages is an
+important step for ensuring that everyone, regardless of their native language,
+can contribute to the project. We are also thrilled that this initiative gives
+end users access to multilingual documentation, making it easier to learn and
+understand OpenTelemetry.
+
+To access the website in your preferred language, use the language
+selector at the top right of the page.
+
+
+We invite you to contribute to this effort. You can find fellow localization
+contributors in the
+[#otel-docs-localization](https://cloud-native.slack.com/archives/C076RUAGP37)
+Slack channel. If you are fluent in a language that is supported already, you
+can help by authoring or reviewing translations. If your language is not yet
+supported and you'd like to help translate, 
+[reach out to us by raising an issue](https://github.com/open-telemetry/opentelemetry.io/issues/new?title=Add+%3Cyour%20language%3E+(%3Cyour+code%3E)+version+of+website+pages&body=%3C!--+please+provide+github+handles+of+at+least+2+people+that+will+work+on+this+translation+project%20--%3E).
```

Comment:
```suggestion
[create an issue](<https://github.com/open-telemetry/opentelemetry.io/issues/new?title=Add+%3CYOUR%20LANGUAGE%3E+(%3CYOUR+LANG+CODE%3E)+version+of+website+pages&body=%3C!--+Provide+GitHub+handles+of+at+least+2+people+that+will+work+on+this+translation+project%20--%3E>).
```

Reply:
Eventually, we could create an issue template that contains the main bullet points of initial things to be covered. WDYT?